src/nvim/tui/tui.c

```diff
@@ -1007,28 +996,35 @@ static void tui_mode_change(UI *ui, String mode, Integer mode_idx)
   data->showing_mode = (ModeShape)mode_idx;
 }
 
-static void tui_grid_scroll(UI *ui, Integer g, Integer top, Integer bot,
-                            Integer left, Integer right,
+static void tui_grid_scroll(UI *ui, Integer g, Integer startrow, Integer endrow,
+                            Integer startcol, Integer endcol,
                             Integer rows, Integer cols)
```

Comment:
@bfredl [pvs noticed](https://neovim.io/doc/reports/pvs/PVS-studio.html.d/sources/tui.c_21.html#ln1015) that `cols` is not used. Should we mark it as false positive?

Reply:
How come it complains on `cols` and not `g` ? Regardless it is a false positive.